038-MdwCovid19/Student/01-Provision.md

```diff
@@ -0,0 +1,67 @@
+# Challenge 1: Provision
+
+[< Previous Challenge](./00-Background.md) - **[Home](../README.md)** - [Next Challenge >](./02-CloudIngest.md)
+
+## Background story
```

Comment:
To follow our template, I think we agreed for consistency that:

Background => Introduction
Technical Details => Description